app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorModel.kt

```diff
@@ -23,4 +24,10 @@ data class SyncIndicatorModel(
   fun syncIndicatorStateChanged(newState: SyncIndicatorState): SyncIndicatorModel {
     return copy(syncIndicatorState = newState)
   }
+
+  fun syncIndicatorSyncedTimerIncremented(newTime: Long): SyncIndicatorModel {
+    val syncedState = syncIndicatorState as Synced
+    val newDurationSince = syncedState.durationSince.plusMinutes(newTime)
```

Comment:
We could change this to a reducer method on `Synced`. 

```kotlin
val newSyncedState = syncedState.plusMinutes(newTime)
```

This way the implementation detail is hidden inside the `Synced` class.